src/components/Answers.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Answers = (props) => {
+  const question = props;
+  if (question) {
+    return (
+      <div>
+        {question.answers.map((question, key) => { return (
```

Comment:
If we are looping through answers, we shouldn't call the current answer a `question`.